frontend/app/controllers/create-badges.js

```diff
@@ -140,20 +140,31 @@ export default Controller.extend({
       }
     },
 
+
     sendBadge(badgeData) {
       const _this = this;
       let badgeRecord = _this.get('store').createRecord('badge', badgeData);
       badgeRecord.save()
         .then(record => {
           _this.set('badgeGenerated', true);
           _this.set('genBadge', record);
-          _this.get('notify').success('Badge generated Successfully');
+          var notify = _this.get('notify');
```

Comment:
@ParthS007 so, after the badges are generated, this notification will appear for some time. So suppose the user is generating a lot of badges (has given a large csv file) and goes away, comes back after some time and within that time the notification has already disappeared. How will the user download the badges then?

Reply:
The my-badges section will always have the links to download them @djmgit

Reply:
That is true, but the user will have to go to my-badges section for downloading incase he misses the notification. Thats becoming a sloppy UX, isn't it?